### ISSUE TITLE
Android not building with react-native 0.47.1

### ISSUE THREAD

**Issue by WolfieZero** [react-native-community/react-native-linear-gradient#202]
Running the project for Android I get the following error

```
{PROJECT DIRECTORY}/node_modules/react-native-linear-gradient/android/src/main/java/com/BV/LinearGradient/LinearGradientPackage.java:20: error: method does not override or implement a method from a supertype
    @Override
    ^
1 error
:react-native-linear-gradient:compileReleaseJavaWithJavac FAILED
```

Seems to be an issue with react-native 0.47.1; downgraded to 0.46.2 and it works.

**Comment by thevishnupradeep** [comment]
Same issue here

**Comment by eagleflo** [comment]
There are two open pull requests to tackle this, #181 and #193. Can we get some maintainer attention here to pick one?

**Comment by scriptfans** [comment]
+1 for this

**Comment by Nickersoft** [comment]
+1

**Comment by dgladkov** [comment]
Fixed in 2.2.0
